mkosi/__init__.py

```diff
@@ -2751,6 +2793,15 @@ def check_tools(config: Config, verb: Verb) -> None:
             check_tool(config, "sbsiglist", reason="set up systemd-boot secure boot auto-enrollment")
             check_tool(config, "sbvarsign", reason="set up systemd-boot secure boot auto-enrollment")
 
+        if ArtifactOutput.pcrs in config.split_artifacts:
+            check_systemd_tool(
+                config,
+                "systemd-measure",
+                "/usr/lib/systemd/systemd-measure",
+                version="257.999",
```

Comment:
Let's not bother with the OBS versions in mkosi, just make this 257

Reply:
But 257 is not right? It will be added in 258, but if I make it 258, then it won't work with packages as the version is 257.999

Reply:
I've added a TODO comment to bump this to 258 once it's released and available